### DISCUSSION TITLE
HTML Text module: Replace Tokens setting resets itself

### DISCUSSION THREAD

**Discussion by DHCuthill** [dnnsoftware/Dnn.Platform#5830] [Q&A] [answered]
Hi
I have just installed DNN PLATFORM v. 09.13.00 (0)
I have a welcome page for those logging in.
This page needs to have the HTML module settings "Replace Tokens" enabled.
This seems to work normally.  Until it doesn't.
After some hours, the page reads as follows
To [User:FIRSTNAME] [User:LASTNAME]
Welcome [User:FIRSTNAME].
You are logged-in as [User:USERNAME]
Upon opening the HTML module settings, it is seen that the "Replace Tokens" checkbox is unticked.

Can anyone cast some light on this problem?

**Comment by Mike35538** [comment, marked as answer]
#5844

**Reply by DHCuthill** [reply]
What is the corrective action I need to do?

**Comment by Mike35538** [comment]
I'm not sure but I'm guessing that you should wait on upgrading until 9.13.1 is released which has the fix in it.

**Reply by DHCuthill** [reply]
Many thanks, Mike.
